styles/globals.css

```diff
@@ -35,3 +36,16 @@ a {
 * {
   box-sizing: border-box;
 }
+
+:root {
+  /* root colors */
+  --MainClr: #890000;
+  --PrimaryClr: white;
+  --InputBgClr: #f2f2f2;
+  --InputTxtClr: black;
+
+  /* root fonts */
+  --BodyFont: "TrajanProRegular", sans-serif;
+  --BodyBoldFont: "TrajanProBold", sans-serif;
+  --StyleFont: "SilliiwillinnRegular", sans-serif;
```

Comment:
PascalCase is pretty unusual for naming CSS variables.
Also don't try to use abbreviations.

Reply:
pascal-case is my coding style... its unusual, but is it bad? greetings